layouts/BeltDocsLayout.re

```diff
@@ -94,23 +94,48 @@ module Md = {
     );
 };
 
+module FlavourSwitch = {
```

Comment:
Probably bike-shedding, but I'd prefer the name `SyntaxSwitch` or `SyntaxToggle` more 😁